add_a_custom_check.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
<p>Fix:</p><p>This guide will demonstrate how to implement a Check with a return value and display, for addding a condition see <a href="https://github.com/deepchecks/deepchecks/raw/683c5ea387dcc0bc0847fb20975a0f0b9e6bf754/link" target="_blank">working with condition</a>s.</p><p> </p><p>and general comment about link</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>